core/app/c/[communitySlug]/stages/manage/components/panel/StagePanelOverviewManagement.tsx

```diff
@@ -13,9 +17,14 @@ export const StagePanelOverviewManagement = (props: Props) => {
 	return (
 		<>
 			<h4 className="mb-2 font-semibold">Stage Management</h4>
-			<Button variant="secondary" onClick={onDeleteClick}>
-				Delete this Stage
-			</Button>
+			<div className="flex gap-2">
+				<Button variant="secondary" asChild>
+					<Link href={`./${props.stageId}`}>Visit this Stage</Link>
```

Comment:
similar to an above comment, i'm shocked that this works, but it does not seem to be documented behavior for next. could we instead change this to `/c/${communitySlug}/stages/${props.stageId}`?